### PR TITLE
WIP: Travis configuration fixes

### PR DESCRIPTION
**Pull request by natebird**
Testing some travis configuration changes to see if it fixes the connection issues.

### CODE REVIEW

**Comment by tanner0101** [comment]
Awesome!! @natebird you've been added to the fluent team. Feel free to merge this whenever.

**Comment by natebird** [comment]
Looks like the OS X build is still failing on Travis. I’ll get that working today. Now that I know what is wrong I should be able to fix it easily enough. :-)

Nate

File: .travis.yml [hunk 1]
```diff
@@ -1,19 +1,14 @@
 os:
   - linux
-  - osx
-language: generic
+  #- osx
```
Comment (tanner0101) [inline .travis.yml:3]: Is the only way to make travis work by commenting out OS X?

Comment (natebird) [reply .travis.yml:3]: I was just narrowing down the issue and wanted to make things complete faster. :-) I'll add it back in.

**Comment by natebird** [comment]
See PR #7 which supersedes this PR.

**Pull request closed without merging**